src/features/experience-matching/ui/company-detail/company-detail.tsx

```
@@ -0,0 +1,73 @@
+    nextStep();
+  };
+
+  return (
+    <div className={styles.layout}>
+      <div className={styles.formWrapper}>
+        {/** 기업 정보 */}
+        <div className={styles.descriptionForm}>
+          {FIELD_CONFIG.map(({ label, key }) => (
+            <div key={key} className={styles.fieldWrapper}>
+              <label className={styles.fieldTitle}>{label}</label>
+              <div className={styles.fieldContent}>{companyData[key]}</div>
```

Biome (2.1.2)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)